src/components/Form/Form.jsx

```diff
@@ -1,20 +1,94 @@
 import React, { useState } from 'react';
 import { useFirebase } from '../../hooks/useFirebase';
+import { useForm } from '../../hooks/useForm';
 
 export const Form = () => {
   const { create } = useFirebase('things');
-  const [thing, setThing] = useState('');
+  const [error, setError] = useState('');
+  const [success, setSuccess] = useState(null);
+
+  const [values, handleInputChange, setValues] = useForm({
+    nameItem: '',
+    selectTime: '',
+    lastDate: null,
+  });
+
   const sendToFB = (e) => {
     e.preventDefault();
-    create({ name: thing });
-    e.target.reset();
+    const itemLength = values.nameItem.length;
+    const selectLength = values.selectTime.length;
+
+    if (itemLength && selectLength) {
+      create({
+        name: values.nameItem,
+        time: values.selectTime,
+        lastDate: null,
+      });
+
+      e.target.reset();
+
+      setValues({
+        nameItem: '',
+        selectTime: '',
+        lastDate: null,
+      });
+      setSuccess('Data was send success');
+      setTimeout(() => {
+        setSuccess(null);
+      }, 2000);
+      return;
+    }
+    setError('Fill in all the blanks');
```

Comment:
Totally optional, especially as it provides a different user experience, but given that you are rightly using a `form` and the `onSubmit` handler, I believe that you can pass `required` props to the `input` fields in the form and html will prevent submission and show a little pop-up with an error message. As I said, it's a different user experience, but just wanted to make sure you were aware so that you can decide which kind of form submission error experience you'd prefer.